### DISCUSSION TITLE
Folder names/titles in Bold

### DISCUSSION THREAD

**Discussion by ricardosteve** [chrisgrieser/shimmering-focus#268] [Q&A] [answered]
Hi, since the newest Shimmering Focus and Obsidian updates, the folders and subfolders in the sidebar have their names in Bold (and when clicked, the Bold is gone, which makes it difficult to distinguish if a subfolder is currently opened / expanded).

I can't find anything on the Style Settings plugin to help me.

My question is:

* Is there a way to make it so that the folders and subfolders, when clicked/expanded, maintain the Bold in the text? How do I do that?

Thanks!

**Comment by chrisgrieser** [comment, marked as answer]
oh that's actually a bug due to the new Obsidian update that I overlooked, since I don't use the file explorer myself. Luckily easy to fix, I just pushed an update that should fix the issue 👍

**Reply by ricardosteve** [reply]
Thanks a lot, this was very helpful! Lovely theme by the way, I couldn't work with Obsidian without it.
